backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/user/api/impl/UserDAO.java

```diff
@@ -18,7 +18,15 @@
 import org.adempiere.model.InterfaceWrapperHelper;
 import org.adempiere.service.ClientId;
 import org.adempiere.util.proxy.Cached;
+import org.compiere.model.I_AD_PInstance;
+import org.compiere.model.I_AD_Preference;
 import org.compiere.model.I_AD_User;
+import org.compiere.model.I_AD_UserMail;
+import org.compiere.model.I_AD_UserQuery;
+import org.compiere.model.I_AD_User_OrgAccess;
+import org.compiere.model.I_AD_User_Roles;
+import org.compiere.model.I_AD_User_Substitute;
+import org.compiere.model.I_C_OrgAssignment;
```

Comment:
pls Organize Imports. I think some of them shall vanish (e.g. I_AD_PInstance, I_AD_Preference etc)

Reply:
cleaned